toolkit/scripts/toolchain.mk

```diff
@@ -97,23 +97,9 @@ copy-toolchain-rpms:
 # check that the manifest files only contain RPMs that could have been generated from toolchain specs.
 check-manifests: check-x86_64-manifests check-aarch64-manifests
 check-aarch64-manifests: $(toolchain_files)
-	cd $(SCRIPTS_DIR)/toolchain && \
-		./check_manifests.sh \
-			"$(toolchain_spec_list)" \
-			"$(SPECS_DIR)" \
```

Comment:
What if any of these values are overridden by the user? I think `SPECS_DIR` is commonly overridden value by product teams.

Reply:
It has the optional parameter "-s" for overriding default value of SPECS_DIR

Reply:
Yes, but the `Makefile` isn't passing it down.